content/Solutions/Optimizations/STIGCompliance.md

```diff
@@ -0,0 +1,273 @@
+---
```

Comment:
Should there be a reference to the FIPS support in this document?  Do we have the FIPS stuff documented yet?

Reply:
We mention FIPS, or FIPS compliance in some of the articles but don't have one article about it generally. I'll add FIPS to this in the next commit